pages/retrospective-assessment/update/js/index.js

```diff
@@ -30,7 +30,8 @@ start({
     grantedVersion: state.static.grantedVersion,
     newApplication: state.static.newApplication,
     sidebarLinks: state.static.sidebarLinks,
-    schemaVersion: 'RA'
+    schemaVersion: 'RA',
+    licenceHolder: state.static.project.licenceHolder
```

Comment:
Does this exist?

Reply:
Yes, `project.licenceHolder` is always the current licence holder for the project, I have not removed this. I believe the RA only applies to the most recent granted version, so this is correct?

What I removed from the API was `version.project.licenceHolder' because when looking at specific versions we should use `version.licenceHolder` instead.